src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_help.py

```diff
@@ -20,6 +23,89 @@
     short-summary: (PREVIEW) Manage Azure IoT hubs.
 """
 
+helps['iot hub certificate'] = """
```

Comment:
Did you go over this verbiage with a PM (Eustace/Chrissie)?

Reply:
I did not, I'll add him.

Reply:
@eustacea